docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Create token credentials from configuration
+description: This article describes how to create Microsoft Entra token credentials from configuration files.
+ms.topic: how-to
+ms.custom: devx-track-dotnet, engagement-fy23
+ms.date: 1/16/2024
+---
+
+# Create Microsoft Entra credential types using configuration files
+
+The `Microsoft.Extensions.Azure` library supports creating different <xref:Azure.Core.TokenCredential?displayProperty=fullName> types from key-value pairs defined in _appsettings.json_ and other configuration files. The credential types correspond to a subset of the [credential classes](/dotnet/api/overview/azure/identity-readme?view=azure-dotnet#credential-classes) in the Azure Identity client library. This article describes the support for different `TokenCredential` types and how to configure the required values for each type.
+
+## Support for Azure credentials through configuration
+
+The [`Microsoft.Extensions.Azure`](https://www.nuget.org/packages/Microsoft.Extensions.Azure) library can automatically provide Azure service clients with a `TokenCredential` class by searching _appsettings.json_ or other configuration files for credential values using the `IConfiguration` abstraction for .NET. This approach allows developers to explicitly set credential values across different environments through configuration rather than through app code directly.
```

Comment:
```suggestion
The [`Microsoft.Extensions.Azure`](https://www.nuget.org/packages/Microsoft.Extensions.Azure) library can automatically provide Azure service clients with a `TokenCredential` class by searching _appsettings.json_, _appsettings.{environment}.json_, or other configuration files for credential values using .NET's `IConfiguration` abstraction. This approach allows developers to explicitly set credential values across different environments through configuration rather than through app code directly.
```